Makefile

```diff
@@ -17,65 +21,56 @@ else
 GOBIN=$(shell go env GOBIN)
 endif
 
-all: hypershift-operator control-plane-operator
+all: build manifests
 
-# Run tests
-test: generate fmt vet manifests
-	go test ./... -coverprofile cover.out
+build: hypershift-operator control-plane-operator
 
-# Build hypershift-operator binary
-hypershift-operator: generate bindata manifests fmt vet bin/hypershift-operator
+verify: build fmt vet
 
-bin/hypershift-operator:
-	go build -o bin/hypershift-operator ./hypershift-operator
+# Generate code
+generate:
+	hack/update-generated-bindata.sh
+	$(CONTROLLER_GEN) object:headerFile="hack/boilerplate.go.txt" paths="./..."
 
-# Build control-plane-operator binary
-control-plane-operator: generate fmt vet
-	go build -o bin/control-plane-operator ./control-plane-operator
+# Build hypershift-operator binary
+hypershift-operator: generate
+	$(GO_BUILD_RECIPE) -o bin/hypershift-operator ./hypershift-operator
 
-# Run against the configured Kubernetes cluster in ~/.kube/config
-run: generate fmt vet manifests
-	go run ./hypershift-operator/main.go
+# Build control-plane-operator binary
+control-plane-operator: generate
+	$(GO_BUILD_RECIPE) -o bin/control-plane-operator ./control-plane-operator
 
-# Install CRDs into a cluster
-install: manifests
-	kustomize build config/crd | kubectl apply -f -
+# Run tests
+test: build
+	$(GO) test ./... -coverprofile cover.out
 
-# Uninstall CRDs from a cluster
-uninstall: manifests
-	kustomize build config/crd | kubectl delete -f -
+# Generate Kube manifests (e.g. CRDs)
+manifests:
+	$(CONTROLLER_GEN) $(CRD_OPTIONS) webhook paths="./..." output:crd:artifacts:config=config/bases/hypershift-operator
 
-# Deploy controller in the configured Kubernetes cluster in ~/.kube/config
-deploy: manifests
-	cd config/manager && kustomize edit set image controller=${IMG}
-	kustomize build config/default | kubectl apply -f -
+# Installs hypershift into a cluster
+install: manifests release-images-data
+	kustomize build config/overlays/$(PROFILE) | kubectl apply -f -
```

Comment:
it'd be nice to have a target to generate manifests but not apply them so we can either inspect them or modify them before applying them.

Reply:
Added a `config` target for this purpose